packages/itwin/reports-config-widget/src/widget/components/Reports.tsx

```diff
@@ -81,6 +80,9 @@ export const Reports = () => {
   const [isLoading, setIsLoading] = useState<boolean>(true);
   const [searchValue, setSearchValue] = useState<string>("");
   const [reports, setReports] = useState<Report[]>([]);
+  const bulkExtractor = useMemo(() =>
+    new BulkExtractor(apiConfig, reports.map((r) => r.id)), [apiConfig, reports]);
```

Comment:
This should be a bit easier to read. Moving dependencies to a separate line makes it easier to find them.
```
const bulkExtractor = useMemo(
  new BulkExtractor(apiConfig, reports.map((r) => r.id)),
  [apiConfig, reports]
);
```

Reply:
Closing parenthesis has been moved to a new line. Dependencies left on the same line due to lint bug regarding indentation bug.

Reply:
Looks good.